src/query/FindArgs.ts

```diff
@@ -1,6 +1,11 @@
-import type { FindOneArgs } from './FindOneArgs';
+import type { FindOneArgs, FindOneArgsTyped } from './FindOneArgs';
 
 export interface FindArgs extends FindOneArgs {
   skip?: number;
   limit?: number;
 }
+
+export interface FindArgsTyped<T> extends FindOneArgsTyped<T> {
```

Comment:
How come the `*Typed<T>` interfaces are needed? Can the original interfaces be adjusted to take a generic param?

Reply:
I originally went down the path of adjusting the original interfaces to support generic types, but it ended up getting really hectic trying to get typings to work with all of the internal columns, etc.. My main goal with this change was to make the public API more type-safe, and I wasn't sure how much benefit there would be to having this internally.